.github/workflows/format-code.yml

```diff
@@ -2,30 +2,31 @@ name: format code
 on:
   pull_request:
     paths:
-      - 'Sources/**'
-      - 'Tests/**'
-      - 'test-server/**'
-      - 'Samples/**'
-      - '.github/workflows/format-code.yml'
+      - "Sources/**"
+      - "Tests/**"
+      - "test-server/**"
+      - "Samples/**"
+      - ".github/workflows/format-code.yml"
 
 jobs:
   # Formats Swift, Objective-C, C++, and C code and commits the formatted code
   # if necessary
   format-code:
     name: Format Code
-    runs-on: macos-13
+    runs-on: macos-15
```

Comment:
Using the opportunity to bump this to macos-15, will require installation of `clang-format`, `swiftlint` and `rbenv` as these are not preinstalled anymore.